metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -223,6 +223,24 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
     "If not set, METAFLOW_NOTIFY_ON_SUCCESS is used from Metaflow config or environment variable",
     show_default=True,
 )
+@click.option(
+    "--sqs-url",
```

Comment:
METAFLOW_SQS_URL --> METAFLOW_DLQ_SQS_URL
--sqs-url --> --dlq-sqs-url

??

Reply:
users can write event messages to any sqs, not have to be a dlq, so prefer to name it more generic.

Reply:
makes sense, however this SQS queue is for failures only?
so to folly the `--notify-on-error` pattern it should be named 
* `METAFLOW_NOTIFY_ON_ERROR_SQS_URL`
* `--notify-on-error-sqs-url`

Reply:
Can you take a look at https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/aip-admission-webhook-service/-/merge_requests/34/ first? Changes here depend on there?

After webhook is approved, I will update accordingly here.

Reply:
updated these names